### ISSUE TITLE
No way of tracking results during fit() function

### ISSUE THREAD

**Issue by OleGunvaldsen** [cair/pyTsetlinMachine#13]
There are currently no way of tracking accuracy, loss etc when running the tm.fit function. This makes the training and testing a black box without any information as to how many epochs is sufficient, how the loss and accuracy develop during training, and how changing the hyperparameters and data format impacts the training.

**Comment by olegranmo** [comment]
Hi Ole! Thanks for the question. I guess you currently have to implement it yourself. Split the training data into smaller pieces, and then do fit with the incremental option on. Then you can track the progress in smaller steps
	tm.fit(small_piece_of_X_train, small_piece_of_Y_train, epochs=1, incremental=True)

To get the accuracy between calling fit on each data chunk, you can have a small test:
	result = 100*(tm.predict(small_X_test) == small_Y_test).mean()
and print out the accuracy.

Would that help solve the problem?
